tests/functional/reporting-non-auth.js

```diff
@@ -286,30 +268,23 @@ define(
           .end();
       },
 
-      "details param adds info to description": function() {
```

Comment:
edit: this test probably *does* need to be modified, since it gets added at a different time. But not deleted.